APMrover2/navigation.cpp

```diff
@@ -22,7 +22,7 @@ void Rover::navigate()
 	wp_distance = get_distance(current_loc, next_WP);
 
 	if (wp_distance < 0){
-		gcs_send_text(MAV_SEVERITY_CRITICAL,"<navigate> WP error - distance < 0");
+		gcs_send_text(MAV_SEVERITY_NOTICE,"<navigate> WP error - distance < 0");
```

Comment:
This is a problem if wp_distance is less then zero.  This function won't stop the rover here but the rover is in a bad way.